plugins/cql-language/src/main/java/org/polypheny/db/cql/CqlQueryBuilder.java

```diff
@@ -194,7 +192,7 @@ public ColumnIndex addColumnIndex( String schemaName, String tableName, String c
 
         String fullyQualifiedColumnName = schemaName + "." + tableName + "." + columnName;
         if ( !columnIndexMapping.containsKey( fullyQualifiedColumnName ) ) {
-            ColumnIndex columnIndex = ColumnIndex.createIndex( databaseName, schemaName, tableName, columnName );
+            ColumnIndex columnIndex = ColumnIndex.createIndex( schemaName, tableName, columnName );
```

Comment:
Should be namespace and entity name

Reply:
Renamed and named whole plugin to use language-unspecific names where appropriate.